src/status_im/contexts/wallet/events.cljs

```diff
@@ -362,18 +362,26 @@
 
 (rf/reg-event-fx :wallet/address-validation-success
  (fn [{:keys [db]}]
-   {:db (assoc-in db [:wallet :ui :search-address :valid-ens-or-address?] true)}))
+   {:db (-> db
+            (assoc-in [:wallet :ui :search-address :valid-ens-or-address?] true)
+            (assoc-in [:wallet :ui :search-address :loading?] false))}))
```

Comment:
This updates the properties within a nested keyword ([:wallet :ui :search-address]) in the db in one step

```suggestion
   {:db (update-in db
                     [:wallet :ui :search-address]
                     assoc
                     :valid-ens-or-address?     true
                     :loading?                  false)}
```

Reply:
Thanks for the suggestion @BalogunofAfrica, updated it!

Reply:
@BalogunofAfrica in this case I think we don't have a strong preference, sometimes when the code is easier to understand or less verbose it's fine, in this case I think it's the same, but it's always good to share different approaches to get the same result :100: